src/Monsters/SpecialRules/AmbushMagic.js

```diff
@@ -28,13 +34,42 @@ export default function (roll) {
                 Magic Ambush is a spellcaster, in the first turn it may cast an extra set of spells as soon as it is
                 placed.
             </p>
```

Comment:
In the unchanged area (I can't select it), should "make on set of attacks" be replaced with "make one set of attacks"?